satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -377,6 +377,102 @@ read limitoffset (
     orderby desc bucket_usage.rollup_end_time
 )
 
+//--- used serials ---//
```

Comment:
this comment is in the wrong place now

Reply:
not really, but the comment does need updating. both tables are necessary for tracking used serials and information

Reply:
hopefully the comment is now clearer